src/views/TechnicalAssistance/RequestForm/AttendeesForm/components.tsx

```diff
@@ -141,7 +141,7 @@ const AttendeeFields = ({
               return (
                 <FormGroup error={!!error}>
                   <Label htmlFor="euaUserId">
-                    {t(fieldLabels.euaUserId)}
+                    {t(fieldLabels.euaUserId || '')}
```

Comment:
purely out of curiosity - why would we translate the user's EUA?

Reply:
Good catch, I think this is maybe a find/replace error on @ClayBenson94 's part. This is not the EUA itself, but a translation for what we label an eua ID as (I think): https://github.com/CMSgov/easi-app/blob/main/src/i18n/en-US/technicalAssistance.ts#L664

Reply:
I wasn't 100% sure on this one. Without this change, you do get a type error, but it was the most cryptic one compared to the others

Reply:
Perhaps the original definition didn't need to get updated to `string | null` here. Lemme try that.

https://github.com/CMSgov/easi-app/blob/f7c09ef3b922d60a146846d2927649e3f3830d62/src/types/technicalAssistance.ts#L25

Reply:
@samoddball @mynar7 Good catch! I pushed a fix for this here. Just removed the type definition and this `||` bit here, and everything still works as expected :)

c772c5988

Reply:
Ahhh thank you both - totally glossed over `fieldLabels`. Sorry!